### ISSUE TITLE
Save extension settings in separate config

### ISSUE THREAD

**Issue by uliska** [frescobaldi/frescobaldi#1219]
I found out that it is possible to create additional settings files with `QSettings(<target>)`.

I should use that for separating extension config from the main app's. However, I'll have to investigate *where* the files are stored (it should *not* be stored in the "current directory") but either in the global config directory or i the extension's directory (worse, I think)). And I have to check how that behaves on Windows.

**Comment by wbsoft** [comment]
It is also maybe nice to add a startup option to run Frescobaldi with all extensions turned off, in the case an extension hinders the startup.

**Comment by uliska** [comment]
That's true. Actually I thought I'd cover this situation, but this issue proved it wasn't sufficient.

When an extension throws an exception while loading a dialog is shown with the option to display the full traceback, and the extension is disabled.

However, in *this* case the extension loaded properly and only caused the error *later* in the process. The extension mechanism properly caught the exception, but it seems that this occured during the GUI initialization and therefore totally crashed the app before one could for example manually disable the extension.

**Comment by uliska** [comment]
I mixed it up, this issue shouldn't have been closed.
